package.json

```diff
@@ -7,14 +7,16 @@
   "main": "dist/index.js",
   "module": "dist/index.js",
   "scripts": {
-    "start": "tsc && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wds\"",
-    "build": "tsc",
-    "prepare": "tsc && husky install",
+    "start": "yarn run prepare && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wds\"",
+    "prepare:ghpages": "rimraf ghpages && yarn run prepare && vite build",
+    "prepare": "rimraf dist && tsc && husky install",
```

Comment:
TY for keeping build the first step so now we are creating review apps & releases off of the same `/dist` directory 👍